samples/System.Devices.Gpio.Samples/Program.cs

```diff
@@ -16,7 +16,8 @@ private enum DeviceKind
             Unknown,
             RaspberryPi = 1,
             Odroid = 2,
-            Hummingboard = 3
+            Hummingboard = 3,
+            RaspberryPiOnWindows = 4,
```

Comment:
nit: Should it be called WindowsOnRaspberryPi?

Reply:
this is basically a test app; I think the name is ok 😄

Reply:
The name is simply an artifact to the way the test program was written. It should take a driver name/number, then a test name, then parameters for the pin number(s).

The only reason I chose RaspberryPiOnWindows is so that I could use a different pin number, so that I can just use the on-board green LED to blink (RPi2 only - not RPi3).